torch/testing/_internal/common_utils.py

```diff
@@ -1795,6 +1795,11 @@ def wrapper(*args, **kwargs):
         return dec_fn(func)
     return dec_fn
 
+def xfailIfXpu(func):
```

Comment:
There is already expectedFailureXPU from torch.testing._internal.common_device_type.